src/controllers/account.ts

```diff
@@ -43,19 +43,19 @@ export async function deleteYourProofOfIdPost(
   if (session) {
     const kvbRDF = await getDatasetUri(
       session,
-      "private/govuk/identity/poc/credentials-pat/vcs/kbv/metadata"
+      "private/govuk/identity/poc/credentials/vcs/kbv/metadata"
```

Comment:
Huh, how come this commit is still here? I thought it would have gone away when you merged https://github.com/alphagov/di-solid-prototype/pull/114

Reply:
I think this was a rebase error, should be gone now